src/daemon/plugin.h

```diff
@@ -34,8 +34,13 @@
 #include "meta_data.h"
 #include "utils_time.h"
 
+#include <inttypes.h>
 #include <pthread.h>
 
+#ifdef WIN32
+#undef ERROR
```

Comment:
can you add a comment as to why you do this?

Reply:
Done.